### DISCUSSION TITLE
Dynamic axis labels style

### DISCUSSION THREAD

**Discussion by smelfungus** [patrykandpatrick/vico#546] [Questions] [answered]
Hello, and thank you for the incredible work on this project. I was wondering if there's a possibility of implementing dynamic styling for axis labels, similar to how `valueFormatter` allows for a dynamic format of values? Any guidance on this would be greatly appreciated.

![image](https://github.com/patrykandpatrick/vico/assets/3989453/059f17d9-1fcb-400f-869f-786ad5569235)

**Comment by patrickmichalik** [comment, marked as answer]
Hello! Thanks for the kind words. You can achieve this via `AxisValueFormatter`. The return type of `formatValue` is `CharSequence`, meaning that `SpannableString`s can be used. For bold, use `StyleSpan(Typeface.BOLD)`.
